src/notification.h

```diff
@@ -12,6 +12,8 @@
 #define NORM 1
 #define CRIT 2
 
+#define MAX_CHARS 5000
```

Comment:
@tsipinakis Is there any naming convention for macros? IMHO `MAX_CHARS` is too generic.